examples/msp432-launchpad-for-ethernet/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = [
+	"9names",
+]
+edition = "2018"
+readme = "README.md"
+name = "msp432-launchpad-for-ethernet"
+version = "0.1.0"
```

Comment:
Is there a licence we can add here?

Reply:
Added licenses to match readme and source material